docs/csharp/language-reference/operators/equality-operators.md

```diff
@@ -79,10 +87,18 @@ The following example demonstrates the usage of the `!=` operator:
 
 A user-defined type can [overload](operator-overloading.md) the `==` and `!=` operators. If a type overloads one of the two operators, it must also overload the other one.
 
+A record type cannot explicitly overload the `==` and `!=` operators. If you need to change the behavior of the `==` and `!=` operators for record type `T`, implement the <xref:System.IEquatable%601.Equals%2A?displayProperty=nameWithType> method with the following signature:
```

Comment:
GetHashCode will also needed to be implemented to avoid CS8851 warning.

Reply:
I think this is too advanced to mention here. Moreover, the warning is clear enough to navigate developers.